package.json

```diff
@@ -62,7 +62,7 @@
     "esformatter-eol-last": "^1.0.0",
     "esformatter-jsx": "^5.0.4",
     "esformatter-quotes": "^1.0.3",
-    "eslint": "~2.2.0",
+    "eslint": "^2.13.*",
```

Comment:
I pinned this to 2.2.0 because I was getting errors during my upgrade PR work. Thank you for updating this.